styles/style.css

```diff
@@ -1,5 +1,254 @@
-* {
+*{
   margin: 0;
   padding: 0;
+}
+
+body {
+  font-family: "Open Sans", sans-serif;
+  background-color: rgb(244, 244, 245);
+}
+.header {
+  margin-left: 12%;
+  margin-right: 12%;
```

Comment:
bunlar yerine `margin-inline: auto` vermelisin ve tasarıma uygun olması için `max-width: 1320px` olarak güncellemelisin